From f219f35225fd71dfcfdd5c93c7470cf705778e5b Mon Sep 17 00:00:00 2001
From: Micah Lee <micah@micahflee.com>
Date: Mon, 11 May 2015 15:27:32 -0700
Subject: [PATCH] Stop letting Tor Browser act as a default browser

---
 share/applications/torbrowser.desktop | 1 -
 1 file changed, 1 deletion(-)

Index: b/share/applications/torbrowser.desktop
===================================================================
--- a/share/applications/torbrowser.desktop	2016-01-15 20:53:02.350748143 +0100
+++ b/share/applications/torbrowser.desktop	2016-01-15 20:53:05.162748239 +0100
@@ -6,4 +6,3 @@
 Type=Application
 Icon=/usr/share/pixmaps/torbrowser80.xpm
 Categories=Network;WebBrowser;
-MimeType=x-scheme-handler/http;x-scheme-handler/https;
